dist/index.js

```diff
@@ -1062,11 +1088,19 @@ function loadConfig() {
         if (!fs_1.default.existsSync(filename)) {
             core.setFailed(`Could not find pipeline at ${baseFolder}/${pipeline}`);
         }
+        core.info(`file parameters ${runtimeParametersFile}`);
```

Comment:
We are trying to minimize the number of info messages output to the user so these ones should be in debug. You can also consolidate both into a single message.